invenio_communities/assets/semantic-ui/js/invenio_communities/requests/requests.js

```diff
@@ -311,38 +310,12 @@ export const RequestsResultsGridItemTemplate = ({ result, community }) => {
   );
 };
 
-const RightBottomLabel = ({ result, className }) => {
```

Comment:
why is this component removed?

Reply:
Originally, the RIghtBottomLabel, was defined both in invenio-app-rdm and invenio-communities, causing code duplication. In a previous PR it was moved to invenio-communities/invenio_communities/assets/semantic-ui/js/invenio_communities/requests/requests_items/RightBottomLabel.js so it could be used by both repositories without duplicating code. This code should have been removed then, but I didn't at the time, so I'm also using this PR to clean it.